gke-dask/notebooks/xgboost-gpu-hpo-job-parallel-k8s.ipynb

```diff
@@ -18,283 +18,20 @@
    "id": "19c0c861-9357-48b1-a317-a9df3deb2319",
```

Comment:
<p>Happy to include this but it's not actually necessary as the cluster object registers a finalizer that will do this when Python exits.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/196/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>